staging/src/k8s.io/apiserver/pkg/server/options/admission_test.go

```diff
@@ -36,7 +36,7 @@ func TestEnabledPluginNames(t *testing.T) {
 	}{
 		// scenario 0: check if a call to enabledPluginNames sets expected values.
 		{
-			expectedPluginNames: []string{"NamespaceLifecycle", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook"},
+			expectedPluginNames: []string{"NamespaceLifecycle", "MutatingAdmissionWebhook", "ValidatingAdmissionWebhook", "CEL"},
```

Comment:
"ValidatingAdmissionPolicy" ? (also, move before validating webhooks here too)

Reply:
This is the plugin name

Reply:
I think maybe Joe is suggesting an alternative plugin name to match the style used by webhook?

Reply:
Thanks. I have update it together with the original plugin name^^